converter/src/test/java/gov/cms/qpp/conversion/encode/JsonWrapperTest.java

```diff
@@ -139,13 +143,58 @@ void testIsObject_false() {
 	}
 
 	@Test
-	void testValidDate() throws Exception {
+	void testValidDateYyyyMmDd() throws Exception {
```

Comment:
Can you please add a couple more tests to verify that the date strings we saw in the service now ticket and the google group will parse correctly?